base_address_parish/README.rst

```diff
@@ -0,0 +1,92 @@
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fpartner--contact-lightgray.png?logo=github
+    :target: https://github.com/OCA/partner-contact/tree/15.0/partner_industry_secondary
+    :alt: OCA/partner-contact
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/partner-contact-15-0/partner-contact-15-0-partner_industry_secondary
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/134/15.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module adds add parish to cities.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+To manage industries, you need to:
+
+* Go to *Contacts > Configuration > Sectors of Activity*.
+
+By default only companies have industries. To activate industries in individuals also,
+you need to activate the following setting:
+
+* Go to *Settings > General Settings > Partner Industries > Industry in contacts > Use industry for individuals*
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Go to any partner's form.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/partner-contact/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/partner-contact/issues/new?body=module:%20partner_industry_secondary%0Aversion:%2015.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* Mamfredy Mejia Matute <@mamfredym>
+
+Contributors
+~~~~~~~~~~~~
+
+* Mamfredy Mejia Matute <@mamfredym>
+
+Maintainers
+~~~~~~~~~~~
+
+This module is maintained by the OCA.
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+This module is part of the `OCA/partner-contact <https://github.com/OCA/partner-contact/tree/15.0/partner_industry_secondary>`_ project on GitHub.
+
+You are welcome to contribute. To learn how please visit https://odoo-community.org/page/Contribute.
```

Comment:
This file is generated by oca-gen-addon-readme, you may asure that there's files called CONTRIBUTRORS.rst and DESCRIPTION.rst on folder readme inside module, you can check here

https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst